packages/p2p/src/components/order-details/order-details.jsx

```diff
@@ -188,26 +212,110 @@ const OrderDetails = observer(({ onPageReturn }) => {
                                     value={payment_info || '-'}
                                 />
                             ))}
-                        <MyProfileSeparatorContainer.Line className='order-details-card--line' />
-                        <OrderInfoBlock
-                            className='order-details-card--padding'
-                            label={
-                                <Text size='xs' weight='bold'>
-                                    {labels.contact_details}
+                        {is_active_order && (
+                            <React.Fragment>
+                                <MyProfileSeparatorContainer.Line className='order-details-card--line' />
+                                <OrderInfoBlock
+                                    className='order-details-card--padding'
+                                    label={
+                                        <Text size='xs' weight='bold'>
+                                            {labels.contact_details}
+                                        </Text>
+                                    }
+                                    value={contact_info || '-'}
+                                />
+                            </React.Fragment>
+                        )}
+                        {is_active_order && (
```

Comment:
Could we not put the two OrderInfoBlock into one block of code instead of having two is_active_order checks? Maybe we could just place both into one block like: and or wrap it with one div?

```
{ is_active_order && (
            <React.Fragment>
                <MyProfileSeparatorContainer.Line className='order-details-card--line' />
                <OrderInfoBlock
                    className='order-details-card--padding'
                    label={
                        <Text size='xs' weight='bold'>
                            {labels.contact_details}
                        </Text>
                    }
                    value={contact_info || '-'}
                />
                <MyProfileSeparatorContainer.Line className='order-details-card--line' />
                <OrderInfoBlock
                    className='order-details-card--padding'
                    label={
                        <Text size='xs' weight='bold'>
                            {labels.instructions}
                        </Text>
                    }
                    value={advert_details.description.trim() || '-'}
                />
            </React.Fragment>
        )
    }
```

Reply:
Can 👍